src/content/selector.js

```diff
@@ -93,17 +103,20 @@ window['pctSelector'] = (function(window) {
     function updateSelect(select, alias, setDefault) {
         var options = select.options;
 
-        for (var i=0; i<options.length; i++) {
-            var option = options[i];
+        var selectDefaultAlias = !isPostPreview() || isReplyPreview();
+        if (selectDefaultAlias) {
```

Comment:
Preference here is to invert the condition and just return early, and save a level of indentation in the rest of the function. (We likely shouldn't run onchange if we don't actually change anything, so there's no "rest of the function" to run.)

Reply:
I thought about a few ways to change this function, trying to keep the code clear and non-redundant. I wanted to minimize repeating the code that sets up the `setDefault` widget. In the existing new-post case, the code needs to initialize `setDefault` to inactive. In the post-preview case which I am fixing, the code needs to initialize `setDefault` depending on whether the user selected their default alias or a different one.

In the end, I decided it would be simplest to just call `select.onchange();` when we are done since the logic is already in there. This is why the loop is wrapped in the new `selectDefaultAlias` condition: to change the selected values if needed, and to run `select.onchange();` afterwards. And this is why the 2 lines are removed: no need to edit `setDefault`'s classList or title, since `onchange()` will take care of it.

I like this approach because it removes redundant logic (The phrase 'This alias is your default for this campaign.' was set from two places before this). Of course, if you prefer an alternate approach, I can change it. :)

Reply:
Ahh, got it -- thanks for explaining. Definitely good with this approach.